README.md

```diff
@@ -51,10 +51,11 @@ If you use `jiant` in academic work, please cite it directly:
 
 ## Papers
 
-`jiant` has been used in these three papers so far:
+`jiant` has been used in these four papers so far:
 
 - [Can You Tell Me How to Get Past Sesame Street? Sentence-Level Pretraining Beyond Language Modeling](https://arxiv.org/abs/1812.10860) (formerly "Looking for ELMo's Friends")
 - [What do you learn from context? Probing for sentence structure in contextualized word representations](https://openreview.net/forum?id=SJzSgnRcKX) ("edge probing")
+- [BERT Rediscovers the Classical NLP Pipeline](https://arxiv.org/abs/1905.05950)
 - [Probing What Different NLP Tasks Teach Machines about Function Word Comprehension](https://arxiv.org/abs/1904.11544) ("function word probing")
 
 To exactly reproduce experiments from [the ELMo's Friends paper](https://arxiv.org/abs/1812.10860) use the [`jsalt-experiments`](https://github.com/jsalt18-sentence-repl/jiant/tree/jsalt-experiments) branch. That will contain a snapshot of the code as of early August, potentially with updated documentation.
```

Comment:
Add some instructions below on where to find scripts?

Reply:
They're not working on master yet; will do once I've tested everything.